src/main/resources/db/migration/V24__normalise_video_link_booking.sql

```diff
@@ -0,0 +1,83 @@
+create table video_link_appointment_pre_dcs_703 as table video_link_appointment;
+create table video_link_booking_pre_dcs_703 as table video_link_booking;
+
```

Comment:
Can't use more compact Postgresql syntax because H2 database doesn't understand it.